qa/vagrant/build.gradle

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+//NOCOMMIT: needs test output formatter
+
+import org.gradle.api.InvalidUserDataException
+
+def testScripts = '2*.bats' //NOCOMMIT this should be *.bats
+def testCommand = "cd \$TESTROOT && sudo bats \$BATS/$testScripts"
+def smokeTestCommand = 'echo I work'
+def representativeBoxes = ['trusty', 'centos-7']
+def boxes = representativeBoxes + ['precise', 'vivid', 'jessie', 'centos-6',
+  'oel-7', 'fedora-22', 'opensuse-13', 'sles-12']
+
+configurations {
+  test
+}
+
+dependencies {
+  test project(path: ':distribution:tar', configuration: 'archives')
+  // NOCOMMMIT: we need the rpm and deb!
+  // test project(path: ':distribution:rpm', configuration: 'archives')
+  // test project(path: ':distribution:deb', configuration: 'archives')
+  // Collect all the plugins
+  for (Project subproj : project.rootProject.subprojects) {
+    if (subproj.path.startsWith(':plugins:')) {
+      test project(subproj.path)
+    }
+  }
+}
+
+task checkPackages {
+  group 'Verification'
+  description 'Check the packages against a representative sample of the ' +
+    'linux distributions we have in our Vagrantfile'
+}
+
+task checkPackagesAllDistros {
+  group 'Verification'
+  description 'Check the packages against all the linux distributions we ' +
+    'have in our Vagrantfile'
+}
+
+task clean {
+  delete buildDir
```

Comment:
I don't think this is what you want...it would delete the build dir every time gradle is configured.... Normally clean is a task of type Delete.